### PR TITLE
IGAPP-789: Fix EventEmitter warnings

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by klinzo** [comment]
No more annoying warnings :tada:

**Review by Taggotty — APPROVED** [review]

File: native/package.json [hunk 1]
```diff
@@ -39,48 +39,49 @@
...
     "@react-navigation/stack": "^6.0.11",
-    "@sentry/react-native": "^3.2.3",
+    "@sentry/react-native": "3.2.3",
     "@turf/distance": "^6.5.0",
     "accordion-collapse-react-native": "^1.1.0",
     "api-client": "0.0.1",
     "build-configs": "0.0.1",
     "events": "^3.2.0",
-    "htmlparser2": "^7.1.2",
+    "fbjs": "^0.8.18",
```
[the remark this answers is not in the document]
Comment (klinzo) [reply native/package.json:62]: Required by react-native-pdf: https://github.com/wonday/react-native-pdf/blob/ad31ee5d5541678ef24ebdc0d30eb20f45ac56f5/README.md#installation :/ However, there is already an issue to remove it: https://github.com/wonday/react-native-pdf/issues/609

**Review by SteffiStuffel — APPROVED** [review]
Tested on web only, as my emulator is on strike.

File: web/src/components/I18nProvider.tsx [hunk 2]
```diff
@@ -73,9 +73,7 @@ const I18nProvider = ({ children, contentLanguage }: PropsType): ReactElement =>
 
   // Apply side effects
   useEffect(() => {
-    if (document.documentElement) {
-      document.documentElement.lang = language
-    }
+    document.documentElement.lang = language
```
[the remark this answers is not in the document]
Comment (klinzo) [reply web/src/components/I18nProvider.tsx]: Typescript told me it is not necessary. Re added the check though, should definitely not be done in a PR like this, you are right :+1:

File: web/src/constants/urls.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
 import buildConfig from './buildConfig'
 
-export const cmsApiBaseUrl = window.localStorage?.getItem('api-url') || buildConfig().cmsUrl
+export const cmsApiBaseUrl = window.localStorage.getItem('api-url') || buildConfig().cmsUrl
```
[the remark this answers is not in the document]
Comment (klinzo) [reply web/src/constants/urls.ts]: Typescript told me it is not necessary. Re added the check though, should definitely not be done in a PR like this, you are right :+1:

File: web/src/index.tsx [hunk 4]
```diff
@@ -23,10 +23,8 @@ if (buildConfig().splashScreen) {
 }
 
 // Currently we do not have service workers. Unregister all previous ones:
-if (navigator.serviceWorker) {
-  navigator.serviceWorker.getRegistrations().then(registrations => {
-    registrations.forEach(registration => {
-      registration.unregister()
-    })
+navigator.serviceWorker.getRegistrations().then(registrations => {
```
[the remark this answers is not in the document]
Comment (klinzo) [reply web/src/index.tsx]: Typescript told me it is not necessary. Re added the check though, should definitely not be done in a PR like this, you are right :+1:

**Review by Taggotty — APPROVED** [review]

**Pull request merged**